Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
 name = "as-witx"
-version = "0.1.0"
+version = "1.2.0"
```

Comment:
Sounds a bit premature. Maybe we should stick with `0.x` for now :)

Reply:
Whoops, good catch! I updated it to `0.20` as originally intended